gap_linux.go

```diff
@@ -11,6 +11,29 @@ import (
 	"github.com/muka/go-bluetooth/bluez/profile/device"
 )
 
+// Address contains a Bluetooth address, which is a MAC address plus some extra
+// information.
+type Address struct {
```

Comment:
I would propose to rename this to something like `MACAddress` and share it between all implementations (in gap.go). That avoids some duplication.

Reply:
OK, can do that.

Reply:
Done.